### ISSUE TITLE
DL html input not sending image information

### ISSUE THREAD

**Issue by perlpty** [voidlabs/mosaico#545]
When Downloading (/dl), In mosaico 0.17.4 html input for image was something like that:
OS = ubuntu 16.04

`<img border="0" hspace="0" vspace="0" width="166" class="mobile-full" alt="" style="border: 0px; display: block; vertical-align: top; width: 100%; height: auto; max-width: 166px;" src="img?src=http%3A%2F%2Fmydomain.com%2Fpromo%2FP68%2Fuploads%2Fimage.jpg&amp;method=resize&amp;params=166%2Cnull">`

So, I could get image information from there when parsing that tag.

However when Downloading (/dl) on mosaico 0.17.5, html input for image is like this:
OS = ubuntu 16.04

`<img border="0" hspace="0" vspace="0" width="166" class="mobile-full mo-preloading" alt="" style="border: 0px; display: block; vertical-align: top; width: 100%; height: auto; max-width: 166px;">`

Is there any change between those versions that might affect html generation when downloading?

**Comment by bago** [comment]
See #514

**Comment by bago** [comment]
Are you sure you are using 0.17.5 and not trunk/HEAD or another non-tagged release?

**Comment by perlpty** [comment]
Hi Bago, I did as recommended in #514:

`	- In wysiwygs.js (src/js/bindings/wysiwygs.js)

	} else {
		element.setArribute('src',src);
	}`

But In my case, that did not solve the trick.  So, I uninstall 0.17.5 and install again the older version I had before: 0.17.4.  That worked.

I didnt test: shutdown server after doing that change, so I cannot confirm if that would have work.  But, what I did see is that I did not have the above line in "wysiwygs.js" file. Also I didnot erase cookies and localstorage either.  Maybe shutting down or deleting cookies/localstorage would have helped but I didnt test that.

Thanks a lot.
